AlphaFold2.ipynb

```diff
@@ -95,18 +93,23 @@
         "# number of models to use\n",
         "use_amber = False #@param {type:\"boolean\"}\n",
         "use_templates = False #@param {type:\"boolean\"}\n",
-        "save_to_google_drive = False #@param {type:\"boolean\"}\n",
-        "#@markdown -  if the save_to_google_drive option was selected, the result zip will be uploaded to your Google Drive\n",
-        "\n",
-        "#@markdown ### Advanced settings\n",
+        "upload_custom_templates = False #@param {type:\"boolean\"}\n",
```

Comment:
`use_templates`, `upload_custom_templates` are confusing labels, its not immediatly clear what the difference is and if you need to have use_templates enabled to use custom templates.

Reply:
I agree. Maybe we could change it to "use_pdb70_template", "use_custom_template"? Would this be easier?

Reply:
I reworked the interface. Its now a drop down box. I also added explanation what the different options mean.